server.js

```diff
@@ -66,15 +66,16 @@ const printWarning = (msg, error) => {
 const method = (m, mw) => (req, res, next) => (req.method === m ? mw(req, res, next) : next());
 
 const app = express()
+  // Load middlewares for parsing JSON, and supporting HTML5 history routing
+  .use(express.json({ limit: '1mb' }))
+  .use(history())
   // Load SSL redirection middleware
   .use(sslServer.middleware)
   // Serves up static files
+  .use('/conf.yml', express.static(path.join(__dirname, 'public/conf.yml')))
+  .use('/extra-conf.js', express.static(path.join(__dirname, 'public/extra-conf.js')))
```

Comment:
Don't think this is needed, because the public directory is already being served up at `/`